base/applications/cmdutils/replace/lang/cs-CZ.rc

```diff
@@ -0,0 +1,53 @@
+/* FILE:       base/shell/cmd/lang/cs-CZ.rc
```

Comment:
Maybe @binarymaster could help you syncing all this file headers :) (see also my other "ReactOS Replace Command" suggestion)

Reply:
Currently handling the headers... please wait.